.buildkite/steps/publish-rpm-package.sh

```diff
@@ -24,12 +24,15 @@ mkdir -p rpm
 buildkite-agent artifact download --build "$artifacts_build" "rpm/*.rpm" rpm/
 
 echo '--- Installing dependencies'
-bundle
+apk --no-cache add createrepo_c --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
+
+# createrepo_c requires some exotic flags on the cp, which aren't available on the busybox version
+apk add coreutils
```

Comment:
Is it safe to assume we have a working APK index here? i.e. `apk update` or whatever has already run?

Reply:
I can it it works in the dry run builds, but I guess that could change at some point. Docker isn't as consistent as I'd like it to be.

Do you think it's worth running `apk update` as a precaution?

Reply:
Probably worth adding, in case the `apk update` in the `docker build` is optimised away in future.